.vimrc

```diff
@@ -202,8 +211,8 @@ noremap L g_
 :let g:coc_global_extensions=[
             \'coc-diagnostic',
             \'coc-marketplace',
-            \'coc-tabnine',
             \'coc-snippets',
+            \'coc-tabnine',
```

Comment:
useful but does it worth it ?  @radyz  what do you think ?

Reply:
I'd keep it. Suggestions work great when no actual intellisense is available